### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by SansPseudoFix**
Not sure if everything is correctly escaped =/

### CODE REVIEW

**Comment by gsantner** [comment]
Thanks for your contribution! ;)
Please use  `<!--   Comment -->` for comments https://stackoverflow.com/a/4597285.
I just saw that other comments in string-*.xml are checked in with // also, I missed that, needs to be changed.

Please rebase/squash the 2 commits than, thanks!

**Comment by SansPseudoFix** [comment]
Mh. Well, I can do it, but I'd always hate rebase/squash stuff, so I hope you have some time ;)

**Comment by SansPseudoFix** [comment]
Oh! I did it! \o/

Right?!

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+        Pour créer une ligne horizontale, utilisez au moins trois ---, ___ ou *** sur une ligne à part. N\'importe quel nombre au-delà de trois aura le même résultat et les espaces entre les caractères n\'ont pas d\'importance.&lt;br> &lt;br>
+
+        &lt;b>Liens intégrés&lt;/b>&lt;br>
+        [texte qui apparaîtra ici](http://lien.adresse.ici \"texte alternatif\") Le \"texte alternatif\" est facultatif et affiché lorsque la souris survole le lien.&lt;br> &lt;br>
+
+        &lt;b>Images&lt;/b>&lt;br>
+        ![Texte alternatif](http://siteweb.com/image.jpg \"titre facultatif\") Le texte alternatif entre crochets est affiché si l\'image ne peut pas être chargée et le titre optionnel est affiché lorsque la souris survole l\'image ; les deux sont utiles mais pas essentiels.&lt;br> &lt;br>
+
+        &lt;b>Escape&lt;/b>&lt;br>
+        Si vous voulez inclure dans votre message un caractère aussi utilisé dans le codage Markdown, vous pouvez empêcher qu\'il soit interprété en l\'\"échappant\". Pour cela, placez un backslash (\) devant le caractère. Vous pouvez toutefois écrire "diaspora*" ou "D*" sans que l\'astérisque devienne un code de formatage !&lt;br> &lt;br>
```
Comment (vanitasvitae) [inline app/src/main/res/values-fr/strings.xml:129]: Mixed escapes and non-escapes. Does this serve any purpose? (I'm relatively new to the project and did not look further into the HTML (?) in the resources.)

**Comment by vanitasvitae** [comment]
:+1:

File: app/src/main/res/values-fr/strings.xml (hunk 1 above)
Comment (gsantner) [inline app/src/main/res/values-fr/strings.xml:129]: String resources with HTML content containing `<` do get parsed and modified by the resource system, and fail on `<` character sometimes

**Comment by vanitasvitae** [comment]
@SansPseudoFix thank you very much :)

**Comment by SansPseudoFix** [comment]
You're welcome. Thank you for the app :)

**Pull request merged**